docs/framework/ui-automation/index.md

```diff
@@ -28,4 +28,6 @@ ms.author: "mhopkins"
  [UI Automation Specification and Community Promise](../../../docs/framework/ui-automation/ui-automation-specification-and-community-promise.md)  
   
 ## Related Sections  
- [Accessibility Samples](https://msdn.microsoft.com/library/ms771315.aspx)
+- [UI Automation Provider Samples](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.5/aa358512(v=vs.90))
```

Comment:
Instead of adding these new links, perhaps it would be best to link to the WPF samples repo with a link text of Accessibility samples: 
https://github.com/Microsoft/WPF-Samples/tree/master/Accessibility